Client/game_sa/CFileLoaderSA.cpp

```diff
@@ -28,6 +28,11 @@ void CFileLoaderSA::StaticSetHooks()
     HookInstall(0x538690, (DWORD)CFileLoader_LoadObjectInstance, 5);
 }
 
+CBuildingSAInterface* CFileLoaderSA::LoadFileObjectInstance(SFileObjectInstance* obj)
```

Comment:
LoadObjectInstance?

Reply:
This is common name for this function. It is used in gta-reverse-modern and  ida ipb

Reply:
yeah i mean it should be LoadObjectInstance instead of LoadFileObjectInstance to match

Reply:
Ah... I see